MekHQ/resources/mekhq/resources/Personnel.properties

```diff
@@ -464,11 +464,11 @@ Phenotype.GENERAL.toolTipText=Error: this should never be displayed and is used
 
 # PrisonerCaptureStyle Enum
 PrisonerCaptureStyle.NONE.text=Disabled
-PrisonerCaptureStyle.NONE.toolTipText=Prisoner Capture is disabled
-PrisonerCaptureStyle.ATB.text=Random
-PrisonerCaptureStyle.ATB.toolTipText=Determine randomly which enemy personnel were captured and which escaped at the end of a battle
-PrisonerCaptureStyle.TAHARQA.text=Automatic
-PrisonerCaptureStyle.TAHARQA.toolTipText=All potentially captured enemy combatants will become prisoners when resolving scenarios.
+PrisonerCaptureStyle.NONE.toolTipText=No enemy personnel is captured at the end of a scenario.
```

Comment:
Should probably be 'Enemy personnel will not be taken prisoner at the end of a scenario;